### PR TITLE
Fix for closing modal on esc press (bug 1165656)

### PR DESCRIPTION
**Pull request by elysium001**
The popup modal for app reviews would not close by esc key.

### CODE REVIEW

**Comment by gurumukhi** [comment]
@ddurst / @ngokevin Please review

File: marketplace-elements.js [hunk 1]
```diff
@@ -415,6 +415,14 @@
                             this.dispatchEvent(MktEvent('mkt-prompt-cancel'));
                             root.dismissModal();
                         });
+
+                        // Closes modal on esc key press.
+                        document.addEventListener('keyup', function(e) {
+                            var key = e.which || e.keyCode;
+                            if (key === 27) {
```
Comment (spasovski) [inline marketplace-elements.js]: We should also trigger the `mkt-prompt-cancel` event like on line 415.

**Comment by elysium001** [comment]
@spasovski, squashed commits into one. Please review.

**Comment by elysium001** [comment]
@ddurst / @ngokevin please review or let me know if I'm missing something.

**Comment by ngokevin** [comment]
Sorry, this must have gotten lost in the mix. Thanks for reminding, this looks good! Thanks.

**Pull request merged**